app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -164,31 +139,28 @@ class GithubWebhook extends RequestHandler<Body> {
 
   /// This method assumes that jobs should be cancelled if they are already
   /// runnning.
-  Future<void> _scheduleIfMergeable(
-    PullRequestEvent event, {
-    @required List<IssueLabel> labels,
-  }) async {
+  Future<void> _scheduleIfMergeable(PullRequest event) async {
     // The mergeable flag may be null. False indicates there's a merge conflict,
     // null indicates unknown. Err on the side of allowing the job to run.
 
     // For flutter/flutter tests need to be optimized before enforcing CQ.
-    if (needsCQLabelList.contains(event.repository.fullName.toLowerCase())) {
-      if (!await _checkForCqLabel(labels)) {
+    if (needsCQLabelList.contains(event.head.repo.fullName.toLowerCase())) {
+      if (!await _checkForCqLabel(event.labels)) {
         return;
       }
     }
 
     // Always cancel running builds so we don't ever schedule duplicates.
     await _cancelLuci(
-      event.repository.name,
+      event.head.repo.name,
```

Comment:
Same question for this property in  all the occurrences in this file, why does head means in this context?